src/index.css

```diff
@@ -1,27 +1,95 @@
-body {
+:root {
+  --primary: #d81e5b;
+  --secondary: #8c3aff;
+  --dark: #131a26;
+  --light: #eee;
+  --arsh: hsl(0, 1%, 56%);
+  --light-arsh: #cecaca;
+}
+
+* {
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Fira Sans', sans-serif;
+}
+
+body {
+  background-color: var(--light);
+}
+
+.App {
+  display: flex;
+  min-height: 100vh;
+  align-items: center;
+  justify-content: center;
+}
+
+.calculator {
+  width: 100%;
+  max-width: 400px;
+  background-color: #fff;
+  overflow: hidden;
+  box-shadow: 0 2px 64px hsla(0, 0%, 0%, 0.2);
+}
+
+.display {
+  padding: 1rem;
+  text-align: right;
+  background-color: var(--arsh);
+  color: var(--light);
+  font-size: 2rem;
+  font-weight: 300;
+}
+
+.display span {
+  font-size: 1rem;
+  color: #888;
+}
+
+button {
+  appearance: none;
+  border: none;
+  outline: none;
+  color: #fff;
+  font-size: 1rem;
+  padding: 1rem;
+  cursor: pointer;
+  transition: 0.3s;
+}
+
+button:hover {
+  opacity: 0.4;
+}
+
+.ops-dig-container {
+  display: flex;
+  flex-direction: row-reverse;
+}
+
+.operators {
+  display: flex;
+  flex-direction: column;
+  width: 7rem;
+}
+
+.operators button {
+  flex: 1 1 0%;
+  background-color: var(--primary);
+  font-weight: 700;
+  border: 1px solid var(--arsh);
+}
```

Comment:
- [ ] As suggested by the previous reviewer, your design must look exactly like the wireframe. T

Reply:
Thank you for the feedback, May I know how exactly my design is not align with the on eprovided, so that i can work on the differences, thank you

Reply:
> Thank you for the feedback, May I know how exactly my design is not align with the on eprovided, so that i can work on the differences, thank you

@Simpleshaikh1, kindly change the background color of your operator buttons from that pink to orange.